src/app/client/src/app/modules/dashboard/components/addusers/addusers.component.ts

```diff
@@ -5044,12 +5044,21 @@ export class AddusersComponent implements OnInit {
     }
 
     let addOrgtempArray: any
+    let roles = [];
```

Comment:
@snehalsabade Is it working?

Reply:
I changed request according to anil, after deployment want to test API changes.